tests/test_dataset.py

```diff
@@ -603,6 +603,40 @@ def test_is_isomorphous(data_unmerged, data_fmodel, sg1, sg2, cell1, cell2):
             assert not result
 
 
+@pytest.mark.parametrize("threshold", [5.0, 1.0, 0.5, 0.1])
+@pytest.mark.parametrize("isomorphous", [True, False])
+@pytest.mark.parametrize("sign", [True, False])  # Which cell is bigger
+def test_is_isomorphous_threshold(threshold, isomorphous, sign):
+    """
+    Test that the DataSet.is_isorphous method's cell_threshold operates
+    on percent difference.
+    """
+    epsilon = 1e-12
+    cell = np.array([34.0, 45.0, 98.0, 90.0, 90.0, 90.0])
+    spacegroup = 19
+
+    ds = rs.DataSet(cell=cell, spacegroup=spacegroup)
+
+    if sign:  # cell_test > cell
+        factor = (200.0 + threshold) / (200.0 - threshold)
+        if isomorphous:  # shrink
+            factor = factor * (1.0 - epsilon)
+        else:  # grow
+            factor = factor * (1.0 + epsilon)
+    else:  # cell_test < cell
+        factor = (200.0 - threshold) / (200.0 + threshold)
+        if isomorphous:  # grow
+            factor = factor * (1.0 + epsilon)
+        else:  # shrink
+            factor = factor * (1.0 - epsilon)
```

Comment:
This test is smart... maybe too smart? It had to read it quite a few times before seeing it clearly.

Generally I like dumb stuff. More like:

```python
if isomorphous:
    cell_change_factor = (100.0 + threshold) / 100.0 - epsilon
else:
    cell_change_factor = (100.0 + threshold) / 100.0 + epsilon

if sign:
    cell_test = cell * cell_change_factor
else:
    cell_test = cell / cell_change_factor
```

or something similar.

Reply:
Okay I think I made it dumber. Let me know.